src/connectors/gsuite_logs.py

```diff
@@ -0,0 +1,139 @@
+"""G Suite Admin Logs
+Collect G Suite API logs using a Service Account or Delegated Credentials
+"""
+
+from googleapiclient.discovery import build
+from google.oauth2 import service_account
+
+from runners.helpers import db
+from runners.helpers.dbconfig import ROLE as SA_ROLE
+
+from .utils import yaml_dump
+
+CONNECTION_OPTIONS = [
+    {
+        'name': 'connection_type',
+        'type': 'select',
+        'options': [
+            {'value': 'login', 'label': "Logins"},
+        ],
+        'title': "Admin Logs Type",
+        'prompt': "The type of G Suite logs you are looking to collect.",
+        'required': True,
+        'default': 'login',
+    },
+    {
+        'name': 'service_user_creds',
+        'type': 'json',
+        'title': "Service User Credentials",
+        'prompt': "JSON Credentials from the Service User",
+        'placeholder': (
+            """{"type": "service_account", "project_id": "my-gcp-project", """
+            """"private_key_id": "34d2d9933e3f4a9ee55b968758e5ca3c2b348221", """
+            """"private_key": "-----BEGIN PRIVATE KEY-----\n...-----END PRIVATE KEY-----\n", """
+            """... }"""
+        ),
+        'required': True,
+        'secret': True,
+    },
+    {
+        'type': 'list',
```

Comment:
should swap 'type' and 'name' here so that the ordering is consistent with the other two